app/ui/src/app/integration/edit-page/current-flow.service.ts

```diff
@@ -32,7 +35,10 @@ export class CurrentFlow {
   private _integration: Integration;
   private _loaded = false;
 
-  constructor(private store: IntegrationStore) {
+  constructor(
+    private store: IntegrationStore,
```

Comment:
Can we rename this to `integrationStore`?  I know the original code was like that but it'd be good to align the instance name to the service class, should be a matter of hitting `F2` in vscode if you use that for your typescript :-)